src/index.js

```diff
@@ -1,10 +1,3 @@
-import "core-js/fn/array/from";
-import "core-js/fn/array/includes";
-import "core-js/fn/map";
-import "core-js/fn/object/get-own-property-symbols";
-import "core-js/fn/object/is";
-import "core-js/fn/object/keys";
-import "core-js/fn/set";
 import Helmet from "./Helmet";
```

Comment:
@cwelch5 @doctyper index.js has been removed, and package.json points to Helmet.js. Also changed the import in the tests. Is that everything that needs to be changed?